dialog.txt

```diff
@@ -36,68 +36,68 @@ Errore di sistema!!!#|
+*Sì, ci andrai?# Fantastico, fantastico# fantastico!#_*Assicurati di prendere# una chiave dal mio# assistente, Abbott.#|
+*Questa chiave aprirà il# cancello dello zoo.# Non perderla.#|
+*Restituiscimela viva,# e sicuramente# sarò un eroe...#_*Ehm, umh, voglio dire# sarai un eroe.#|
+*Perché se non è il# nostro eroe ¹£@,#_*Hai dello sterco di# tigre sui tuoi# vestiti?#_*Ha-ha, sto solo# scherzando. Adesso vai# ragazzo.#|
+*Ho sentito delle# voci su una# ragazza prodigiosa.#_*Ti aiuterà# nel tuo viaggio.#|
+*Le chiavi dello Zoo# te le tirano dietro,# prendine un'altra.#|
+*Dai un'occhiata,# tutti gli animali# sono scappati.#|
+*È rimasto solo questo# pulcino di canarino.# Vuoi comprarlo?#|
+*Uhm, che ne dici di# $³ª|§|?#|
+*OK ragazzo, siamo# in un paese libero,# no?#|
+*Prenditene cura.# Torna a trovarci!#|
+*Se è gratis,# lo vuoi?# E prenditelo!#|
+*Come sta il canarino?#|
+*Credo che qualcosa# stia controllando# gli animali.#|
+*Ah... che animaletti# adorabili, vero?#|
+*Mmmm.#|
+*Pezzo di idiota!#|
+*Come posso aiutarti?# Hai qualcosa# da restituire?#|
+*Aspetta.. Una voce sta# parlando nella# mente di ¹£@.#|
+*Bene.#|
+*Grazie per# i vostri acquisti# con noi.#|
+*Com'è comodo che# lo sportello Bancomat#_*sia qui accanto# al bancone, ma perché#_*dovrebbe interessarmi?# Non ho nemmeno una# carta da usare.#|
+*Oggi il cielo# è sereno.# Però, non m'importa.#|
+*Ooh! Quel canarino# è il pulcino di# Laura cantante.#_*Le ridarai il# suo piccolo?#|
+*Momento...#_*Ti sei appena perso# qualcosa di molto# importante.#|
+*Dov'è Laura?...#|
+*Laura si ricordò# la sua canzone, ed iniziò# a cantarla nuovamente.#_*Hai appreso# la melodia?#|
+*Non ci siamo.#|
+*Il fatto è che...# c'è del potere# nella canzone.#|
+*Ciao! Io sono Pippi,# e mi sono smarrita.#_*Sei stato molto# coraggioso a# ritrovarmi.#|
+*Non darti tutte# queste arie.# Ha-ha-ha.#|
+*Sei coraggioso.# speriamo di# incontrarci ancora.#|
+*Sono così felice!  Per# ringraziarti prendi# questo regalo.#_*Un emblema che# ti proteggerà.# Ecco, tieni!#|
+*Probabilmente# hai già# qualcuno di speciale,#|
+*È giunta l'ora# di salutarci.#_*Passa a trovarmi# qualche volta, non# vedo l'ora#_*di rivederti.# Ad ogni modo...#_*Pensi che io# debba rifarmi# l'acconciatura?#|
+*YNon serviva che# tu rispondessi, tesoro.#|
 *EEE, EE EEK,# EEK EEK!#|
-*¹£@ felt this# question enter # his mind.#_*”Who has lost# his tail?”#|
-*My! What strange# clothing you wear!#_*Surely monsters# will come after# those clothes!!#|
-*Are you bothered that# unhappiness and#_*misfortune search# you out?#|
-*If you desire to# never find# trouble...#_*STAY HOME!!#|
+*¹£@ ha sentito questa# questa domanda# nella sua mente.#_*”Chi si è perso# la sua coda?”#|
+*Cielo! che strani# vestiti che indossi!#_*Di certo i mostri# daranno la caccia# a questi indumenti!!#|
+*Ti disturba che# la tristezza#_*e la disgrazia ti# cerchino di continuo?#|
+*Se non desideri# trovarti mai# nei guai...#_*RESTA A CASA!!#|

```

Comment:
Cambierei in `*Se desideri di non# trovarti mai# nei guai...#_*RESTATENE A CASA!!#|`.